integration/fixtures.py

```diff
@@ -127,3 +127,10 @@ def mr_in(ds, mr_alias, subvars):
             'column': [subvariables[subvar_alias(mr_alias, sv)].id for sv in subvars]
         }]
     }
+
+
+def as_entity(body):
```

Comment:
I moved this one here because I was having a warning for the lambda that should not be assigned. That said, We have the same function here in pycrunch https://github.com/Crunch-io/pycrunch/blob/9bb073368293f1e84843a53bdc0cfccd1c90787e/src/pycrunch/shoji.py#L25. Should we use that one here?